x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -40,7 +39,7 @@ const navigationTree: NavigationTreeDefinition = {
           title: i18n.translate('xpack.serverlessSearch.nav.devTools', {
             defaultMessage: 'Dev Tools',
           }),
-          children: getPresets('devtools').children[0].children,
+          children: [{ link: 'dev_tools:console' }, { link: 'dev_tools:searchprofiler' }],
```

Comment:
This is good but do those pages still have the tabs that show the other options?